ks3-test/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+   id("ks3.conventions.lang.kotlin-multiplatform")
```

Comment:
Perhaps the test-utils package can be published, so others can use it to test their serializers.

Reply:
True. I'll put that back